model/client4.go

```diff
@@ -2912,6 +2916,16 @@ func (c *Client4) TestEmail(config *Config) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// TestSiteURL will attempt to connect to the configured SMTP server.
+func (c *Client4) TestSiteURL(siteURL string) (bool, *Response) {
+	r, err := c.DoApiPost(c.GetTestSiteURLRoute(), siteURL)
```

Comment:
I don't see we do it this way.  If in the request body, we usually map it into JSON and explicitly name the element so it's like ``{"site_url":"http://localhost:8065"}``